app/experimenter/nimbus-ui/src/lib/mocks.tsx

```diff
@@ -312,11 +306,20 @@ export function mockExperimentQuery<
       enrollmentEndDate: null,
     },
     modifications,
-  );
+  ) as T;
+}
 
-  if (modifications === null) {
-    experiment = null;
-  }
+export function mockExperimentQuery<
+  T extends getExperiment["experimentBySlug"] = getExperiment_experimentBySlug
+>(
+  slug = "foo",
```

Comment:
Forgot to note this, but I'm also slowly moving us away from needing to specify a slug when creating a new mockedExperimentQuery. In most cases a default slug is fine, and only when we're testing routes and some other minor areas do we need to control the slug.